cocos/base/CCDirector.cpp

```diff
@@ -210,6 +216,13 @@ Director::~Director(void)
     
     Configuration::destroyInstance();
 
+    Sprite3DMaterial::releaseBuiltInMaterial();
+
+    cocostudio::TransformHelp::release();
+
+    ObjectFactory::destroyInstance();
```

Comment:
Got it, I removed them from `Director` in the latest commit. Let me know if there's anything else that needs to be done -- I'll work on it.